cmake/DownloadCMakeLists.txt.in

```diff
@@ -17,8 +17,6 @@ ExternalProject_Add(${package_name}
                "-DCMAKE_AR=${CMAKE_AR}"
                "-DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}"
```

Comment:
I get warnings for `CMAKE_C_COMPILER` and `CMAKE_C_FLAGS`  for  `gflags` and `gtest`. Maybe you can remove these flags for these two libraries.

Reply:
I am not entirely sure about that, they claim that [they need the C language](https://github.com/google/googletest/blob/df428ec11891f12c81e2872c0432e342b5403a34/googletest/CMakeLists.txt#L42)

Reply:
okay, i guess we can leave it in.